constraint/Makefile

```diff
@@ -33,16 +33,8 @@ manifests: buildutils
 	controller-gen all
 	kustomize build config -o deploy/crds.yaml
 
-# Run go fmt against code
-fmt:
-	go fmt ./pkg/...
-
-# Run go vet against code
-vet:
-	go vet ./pkg/...
-
 lint:
-	golangci-lint -v run ./... --timeout 5m --skip-files '(^|/)zz_generated'
```

Comment:
do the generated files now comply with the linter?

Reply:
seems fine https://github.com/sozercan/frameworks/runs/1383770406?check_suite_focus=true